File: frontend/Models/DNHACUNGCAP.cs
using System;

namespace QuanLyBar.Client.Models
{
    public class DNHACUNGCAP
    {
        public int? Id { get; set; }
        public string Name { get; set; }
        public string Note { get; set; }
        public int? DnhomnhacungcapId { get; set; }
        public bool? Status { get; set; }
        public int? UsermodifiedId { get; set; }
        public DateTime? Timemodified { get; set; }
        public DateTime? Timecreated { get; set; }
        public int? UsercreatedId { get; set; }
        public string Manhacungcap { get; set; }
        public decimal? Diachi { get; set; }
        public string Dienthoai { get; set; }
        public string Email { get; set; }
        public string Website { get; set; }
    }
}

File: backend/src/models/DNHACUNGCAP.ts
import { Model, DataTypes } from 'sequelize';
import db from '../config/database';

class DNHACUNGCAP extends Model {
    public id!: any;
    public name!: any;
    public note!: any;
    public dnhomnhacungcapid!: any;
    public status!: any;
    public usermodifiedid!: any;
    public timemodified!: any;
    public timecreated!: any;
    public usercreatedid!: any;
    public manhacungcap!: any;
    public diachi!: any;
    public dienthoai!: any;
    public email!: any;
    public website!: any;
}

DNHACUNGCAP.init({
    ID: {
        type: DataTypes.INTEGER,
        primaryKey: true, autoIncrement: true,
    },
    NAME: {
        type: DataTypes.STRING,
        
    },
    NOTE: {
        type: DataTypes.STRING,
        
    },
    DNHOMNHACUNGCAPID: {
        type: DataTypes.INTEGER,
        
    },
    STATUS: {
        type: DataTypes.BOOLEAN,
        
    },
    USERMODIFIEDID: {
        type: DataTypes.INTEGER,
        
    },
    TIMEMODIFIED: {
        type: DataTypes.DATE,
        
    },
    TIMECREATED: {
        type: DataTypes.DATE,
        
    },
    USERCREATEDID: {
        type: DataTypes.INTEGER,
        
    },
    MANHACUNGCAP: {
        type: DataTypes.STRING,
        
    },
    DIACHI: {
        type: DataTypes.DECIMAL(18,2),
        
    },
    DIENTHOAI: {
        type: DataTypes.STRING,
        
    },
    EMAIL: {
        type: DataTypes.STRING,
        
    },
    WEBSITE: {
        type: DataTypes.STRING,
        
    },
}, {
    sequelize: db.sequelize,
    modelName: 'DNHACUNGCAP',
    tableName: 'DNHACUNGCAP',
    timestamps: false
});

export default DNHACUNGCAP;
